src/discovery/azure/Akka.Discovery.Azure/AkkaHostingExtensions.cs

```diff
@@ -123,11 +128,17 @@ public static class AkkaHostingExtensions
             TableClientOptions? tableClientOptions = null,
             string? serviceName = null,
             string? publicHostname = null,
-            int? publicPort = null)
+            int? publicPort = null,
+            string discoveryId = "azure",
+            bool? readOnly = null,
+            bool isDefaultPlugin = true)
```

Comment:
An idea: what if we just used the `service-name` to alias the HOCON settings, so there's only one moving thing that needs to be tracked? This way, we don't even need to worry about default plugins et al.

Reply:
That works, but that will break previous behavior.

Reply:
That would probably be ok as long as we had some way of covering the default case from before